nifi-registry-core/nifi-registry-security-utils/src/main/java/org/apache/nifi/registry/security/util/KeyStoreUtils.java

```diff
@@ -65,18 +73,4 @@ public static KeyStore getKeyStore(String keyStoreType) throws KeyStoreException
         }
         return KeyStore.getInstance(keyStoreType);
     }
-
-    /**
-     * Returns an empty KeyStore intended for use as a TrustStore backed by the appropriate provider
-     *
-     * @param trustStoreType the trustStoreType
-     * @return an empty KeyStore
-     * @throws KeyStoreException if a KeyStore of the given type cannot be instantiated
-     */
-    public static KeyStore getTrustStore(String trustStoreType) throws KeyStoreException {
```

Comment:
Any reason for removing this/removing the deprecation warning?

Reply:
Thanks for reviewing.  The deprecation warning is not accurate, as PKCS12 is the default key store type starting with Java 9, so the warning and this method are no longer necessary.